src/webhook/signatureEd25519.ts

```diff
@@ -1,17 +1,20 @@
 import { config } from "../config.js";
 import { toText } from "../fetch/cors.js";
-import { verify } from "./verify.js";
+import { Err, Ok, Result } from "../result.js";
+import { CachedVerifier } from "./cached-verifier.js";
 
-export default async function (req: Request, text: string) {
-  const timestamp = req.headers.get("x-signature-timestamp");
+let verifier: CachedVerifier | null = null;
```

Comment:
why can this be `null` ?

`verifier` could be defined in `configs.ts`

Reply:
It was because it is not initialized at first. Will move to `config.ts`.